frontend/components/events/create/DescriptionImageForm.tsx

```diff
@@ -19,12 +20,38 @@ export function DescriptionImageForm({
   setImagePreviewUrl,
   updateField,
 }: DescriptionImageFormProps) {
+  const [previousImages, setPreviousImages] = useState<string[]>([]);
```

Comment:
is this based off reggies changes? is this being merged into master or into reggie's branch?